src/news.js

```diff
@@ -0,0 +1,50 @@
+class News {
+  constructor() {
+    this.date = this.getDate()
+  }
+  getDate() {
+    let today = new Date();
+    let day = String(today.getDate()).padStart(2, '0');
+    let month = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
+    let year = today.getFullYear();
+    today = `${year}-${month}-${day}`
+    return today
+  }
+  connectAPI() {
+    let url = "https://content.guardianapis.com/search?"
+    let tag = "&tag=politics/politics"
+    let date = this.getDate()
+    let api = "&api-key=be4ccfea-6ee5-4d67-aa1e-2d8c1a87574e"
```

Comment:
It's dangerous to put API keys directly into code you upload to github (or anywhere else). Why do you think that might be and what are ways that you could avoid doing this?